<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-02-10 10:15:07.570 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:15:07.570364 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.247308.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1983}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1944}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-02-10 10:15:07.570 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:15:07.570773 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.247315.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/3 (gen_server.erl, line 2435)
    ancestors: [<0.247308.0>]
    message_queue_len: 0
    messages: []
    links: [<0.247308.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 588
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-02-10 10:15:47.904 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:15:47.903935 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.251659.0&gt;,
                               {#Ref&lt;0.3565958295.847773697.241581&gt;,1003993,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.3565958295.847773697.241581&gt;,1003993,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-02-10 10:15:47.904 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:15:47.904208 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.251679.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3565958295.847773697.241581&gt;,1003993,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [&lt;0.251659.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 413
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2025-02-10 10:20:31.190 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:20:31.190371 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.257499.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3565958295.849346561.47008&gt;,6093,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1696)
    ancestors: [&lt;0.257497.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28119
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-02-10 10:20:40.497 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:20:40.497539 ===
** Generic server <0.258754.0> terminating 
** Last message in was {nodedown,'peer_down_crash-501186-11900@058c2d7e9720'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-501186-11900",
                              peer_down => crash},
                            'peer_down_crash-501186-11900@058c2d7e9720',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,46,50,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-501186-11900",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDA1OGMyZDdlOTcyMAAD8sIAAAAAZ6nBBQ=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-501186-11900@058c2d7e9720'}},
                            {<0.258752.0>,#Ref<0.3565958295.849346561.63954>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-501186-11900@058c2d7e9720'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2025-02-10 10:20:40.498 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:20:40.497863 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258754.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-501186-11900@058c2d7e9720'}
      in function  gen_server:handle_common_reply/5 (gen_server.erl, line 2483)
    ancestors: [<0.258752.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1495
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-02-10 10:20:41.496 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:40.496175 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-501378-11900@058c2d7e9720' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-02-10 10:20:41.939 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:41.939341 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-501410-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-02-10 10:20:42.021 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:42.021311 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-501346-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-02-10 10:20:57.523 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:56.522644 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-439011-11900@058c2d7e9720' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-02-10 10:20:58.015 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:58.015319 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-443012-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-02-10 10:20:58.040 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:20:58.040223 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-371973-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:00.262 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:21:00.262160 ===
** Generic server <0.259135.0> terminating 
** Last message in was {'EXIT',<0.259132.0>,
                               {#Ref<0.3565958295.842006530.195602>,1650013,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-439555-11900",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-439555-11900@058c2d7e9720',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,46,50,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-439555-11900",
                                     "-user","peer"],
                                    #Port<0.3129>,undefined,<<>>,running,
                                    {<0.259132.0>,
                                     #Ref<0.3565958295.842006530.195705>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3565958295.842006530.195602>,1650013,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:00.262 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:21:00.262499 ===
  crasher:
    initial call: peer:init/1
    pid: <0.259135.0>
    registered_name: []
    exception exit: {#Ref<0.3565958295.842006530.195602>,1650013,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [<0.259132.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3129>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4592
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:00.819 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:21:00.818739 ===
** Generic server <0.259173.0> terminating 
** Last message in was {'EXIT',<0.259137.0>,
                           {#Ref<0.3565958295.841482244.135255>,2197089,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "YBZNDJPSISCXUNFUAELP","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@058c2d7e9720.2025-02-10_10.04.04/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-443236-11900",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@058c2d7e9720.2025-02-10_10.04.04/make_test_dir.stdlib_test.logs/run.2025-02-10_10.04.41/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-443236-11900",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.7/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-443236-11900@058c2d7e9720',
                                    "/usr/local/lib/erlang-26.2.5.7/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-443236-11900",
                                     "-setcookie","YBZNDJPSISCXUNFUAELP",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@058c2d7e9720.2025-02-10_10.04.04/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-443236-11900",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@058c2d7e9720.2025-02-10_10.04.04/make_test_dir.stdlib_test.logs/run.2025-02-10_10.04.41/log_private/",
                                     "-user","peer"],
                                    #Port<0.3156>,undefined,<<>>,running,
                                    {<0.259137.0>,
                                     #Ref<0.3565958295.841744387.65561>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3565958295.841482244.135255>,2197089,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:00.819 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:21:00.819179 ===
  crasher:
    initial call: peer:init/1
    pid: <0.259173.0>
    registered_name: []
    exception exit: {#Ref<0.3565958295.841482244.135255>,2197089,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [<0.259137.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3156>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4315
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:01.234 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:21:00.233474 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-501794-11900@058c2d7e9720' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:01.372 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:21:01.372311 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-439491-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:01.644 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:21:01.644357 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-501762-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-02-10 10:21:01.651 ***</b>
=WARNING REPORT==== 10-Feb-2025::10:21:01.651165 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-501890-11900@058c2d7e9720'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-02-10 10:42:18.605 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:42:18.604739 ===
** Generic server <0.336161.0> terminating 
** Last message in was {'EXIT',<0.336159.0>,
                               {#Ref<0.3565958295.881852417.92281>,350711,
                                [{child1,<0.336168.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.336161.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.336167.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458459],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3565958295.881852417.92281>,350711,
    [{child1,<0.336168.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-02-10 10:42:18.605 ***</b>
=ERROR REPORT==== 10-Feb-2025::10:42:18.604740 ===
** Generic server <0.336162.0> terminating 
** Last message in was {'EXIT',<0.336159.0>,
                               {#Ref<0.3565958295.881852417.92281>,350711,
                                [{child1,<0.336168.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.336162.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.336168.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458459],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3565958295.881852417.92281>,350711,
    [{child1,<0.336168.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-02-10 10:42:18.605 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:42:18.605044 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.336161.0>
    registered_name: []
    exception exit: {#Ref<0.3565958295.881852417.92281>,350711,
                     [{child1,<0.336168.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [<0.336159.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1013
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-02-10 10:42:18.605 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:42:18.605208 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.336162.0>
    registered_name: []
    exception exit: {#Ref<0.3565958295.881852417.92281>,350711,
                     [{child1,<0.336168.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [<0.336159.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1030
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2025-02-10 10:43:02.108 ***</b>
=CRASH REPORT==== 10-Feb-2025::10:43:02.108563 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.436331.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/6 (gen_server.erl, line 2308)
    ancestors: [bridge9212,<0.436326.0>]
    message_queue_len: 0
    messages: []
    links: [<0.436332.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 354
  neighbours:
    neighbour:
      pid: <0.436332.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.436331.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2025-02-10 11:08:22.773 ***</b>
=WARNING REPORT==== 10-Feb-2025::11:08:22.771175 ===
Process <0.439864.0> on node 'test_server@058c2d7e9720' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2025 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Mon Feb 10 2025 10:04:41<!--/date--><br />
</div>
</center>
</body>
</html>
